view/frontend/web/js/fotorama-video-events-mixin.js

```diff
@@ -14,29 +15,9 @@ define([
                 }
 
                 const videoElement = event.target.querySelector('.product-video');
```

Comment:
I didn't see this before, but it would be better to use `e` instead of `event`, right? Also, probably we can inline this?